server/publications/collections/orders.js

```diff
@@ -17,6 +17,13 @@ Meteor.publish("Orders", function () {
     return Orders.find({
       shopId: shopId
     });
+  } else if (Reaction.getVendorId(this.userId)) {
+    const allOrders = Orders.find({ items:
+    { $elemMatch:
+        { shopId: Reaction.getVendorId(this.userId)}
```

Comment:
Assign `Reaction.getVendorId(this.userId)` to a variable to avoid calling ithe method multiple times.